views/partials/comment.handlebars

```diff
@@ -1,9 +1,7 @@
-<div class="mui--text-body2">
-  {{comment}}
-</div>
-<div class="mui--text-caption">
-  {{user_id}}
-  *
-  {{format_date updatedAt}}
-</div>
-<div class="mui-divider"></div>
\ No newline at end of file
+<div class="card bg-secondary mb-3">
```

Comment:
`bg-secondary` makes the text a bit too hard to read. I think `bg-light` would be more appropriate here.